### PR TITLE
KS Emoticon Glyphs 5.4, part ii

### PR DESCRIPTION
**Pull request by alex40724**
Second part of
https://github.com/ILIAS-eLearning/ILIAS/pull/908

Feature Request:
https://www.ilias.de/docu/goto_docu_wiki_wpage_4698_1357.html

Rules:
https://github.com/ILIAS-eLearning/ILIAS/pull/926/files

### CODE REVIEW

**Comment by alex40724** [comment]
@Amstutz I included the monochrome emoji font suggested by @klees in #908 (see trunk, already merged). However I would be happy, if you and Yvonne could improve the glyphs. We only need a much smaller character set and the glyphs we are using do not work very well in smaller sizes.

**Comment by Amstutz** [comment]
@alex40724 thanks for the notification. We are on to it. However, we currently have a quite busy schedule until the end of april. So get back to us, if you need results before the beginning of mai. In my mind, this is not urgent, but must be done before beta.

**Comment by alex40724** [comment]
@Amstutz No hurry, I just wanted to let you know that monochrome/hovering already works with this set.

**Review by Amstutz — APPROVED** [review]
Looks good to me. Did not run the tests, but the examples and the code as well as the interface do look fine.

File: src/UI/Component/Glyph/Factory.php [hunk 1]
```diff
@@ -550,4 +550,130 @@ public function like($action = null);
...
+	 * @param	string|null	$action
+	 * @return	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function wow($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       Clicking the Sad Glyph indicates a user finds an item discouraging, e.g. a posting.
```
Comment (atoedt) [inline src/UI/Component/Glyph/Factory.php]: disconcerting

Comment (matthiaskunkel) [reply src/UI/Component/Glyph/Factory.php]: JF: accepted

File: src/UI/Component/Glyph/Factory.php [hunk 2]
```diff
@@ -550,4 +550,130 @@ public function like($action = null);
...
+	 * @param	string|null	$action
+	 * @return	\ILIAS\UI\Component\Glyph\Glyph
+	 */
+	public function laugh($action = null);
+
+	/**
+	 * ---
+	 * description:
+	 *   purpose: >
+	 *       Clicking the Wow Glyph indicates a user finds an item surprising, e.g. a posting.
```
Comment (matthiaskunkel) [inline src/UI/Component/Glyph/Factory.php]: glyph should be renamed to 'astounded'

**Comment by matthiaskunkel** [comment]
Jour Fixe, 26 MAR 2018 : PR is highly appreciated. Please respect latest suggestions and then commit to trunk.

**Pull request merged**